core/lib/spree/app_configuration.rb

```diff
@@ -444,7 +444,7 @@ def payment_canceller
     # @return [Module] a module that can be included into Spree::Image to allow attachments
     # Enumerable of images adhering to the present_image_class interface
     class_name_attribute :image_attachment_module, default: 'Spree::Image::ActiveStorageAttachment'
-
+    class_name_attribute :allowed_image_mime_types, default: %w(image/jpeg image/jpg image/png image/gif).freeze
```

Comment:
Can you please add some documentation to this configuration as the other ones?

Reply:
Implemented!